index.d.ts

```diff
@@ -30,6 +33,11 @@ export interface Options {
 	 * The registry URL is by default inferred from the npm defaults and `.npmrc`. This is beneficial as `package-json` and any project using it will work just like npm. This option is **only** intended for internal tools. You should **not** use this option in reusable packages. Prefer just using `.npmrc` whenever possible.
 	 */
 	readonly registryUrl?: string;
+
+	/**
+	 * Overwrite the `agent` that is passed down to [`got`](https://github.com/sindresorhus/got#agent). This might be useful to add [proxy support](https://github.com/sindresorhus/got#proxies).
+	 */
+	readonly agent?: http.Agent | boolean | AgentOptions;
```

Comment:
I think the Got types are incorrect. It should be:
```suggestion
	readonly agent?: HttpAgent | HttpsAgent | boolean | AgentOptions;
```